.travis.yml

```diff
@@ -55,6 +57,11 @@ before_script:
 script:
   - ./lint.sh
   - nosetests -sv test --with-coverage --cover-package=mhctools
+  # run vaxrank unit tests as a general integration test to make sure the stack isn't broken
+  - pip install -r $PWD/vaxrank/requirements.txt
+  - pyensembl install --release 87 --species human
+  - pyensembl install --release 87 --species mouse
+  - travis_wait nosetests test -a '!skip' $PWD/vaxrank
```

Comment:
Have had issues with `travis_wait` not doing what it says it does in the past; what is the goal here?